packages/components/pages/f-offers/src/components/NoOffersFound.vue

```diff
@@ -88,16 +93,45 @@ export default {
     & .c-noOffersFound-searchBox {
         padding-top: spacing(d);
 
+        & input {
+            background-color: $color-white;
+            height: 61px;
```

Comment:
One minor thing, it might be nice to extract the hard-coded `height`s in the styles into variables so that you can assign a descriptive name which will help others to understand what they are used for.

Reply:
Have added variables indicating that they are height adjustments to make it look correct at certain sizes